client/me/chat/index.jsx

```diff
@@ -0,0 +1,25 @@
+/*
+	External Deps
+*/
+import React from 'react';
+import page from 'page';
+
+/*
+	Internal deps
+*/
+import { renderWithReduxStore } from 'lib/react-helpers';
+import controller from 'me/controller';
+import LiveChat from 'components/happychat/page';
```

Comment:
This `page` subcomponent should be moved to this folder (`/me/chat/`) as `main.jsx`.